discovery_engine_flutter/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,6 @@
     package="com.example.xayn_discovery_engine_flutter_example">
    <application
         android:label="xayn_discovery_engine_flutter_example"
-        android:name="xayn_discovery_engine_flutter_example"
```

Comment:
FYI @acrrd - I had to remove this again (added by you in 6557e09aa56b5a73d374efbcf75de8704ad130c2). The app builds when this is present, but crashes on launch because it tries to resolve `android:name` as a class.